cruise.umple/src/StateMachine_Code.ump

```diff
@@ -553,7 +553,7 @@ class State
 
   public String getType()
   {
-    return numberOfTransitions() == 0 && numberOfNestedStateMachines() == 0 ? "Simple" : "Complex";
+    return numberOfTransitions() == 0 && numberOfNestedStateMachines() == 0 && numberOfActions() == 0 ? "Simple" : "Complex";
```

Comment:
why should the number of actions affect the type of states? is the type `Complex` different than the type `composite`?